charts/Chart.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
+- name: localpv-provisioner
+  repository: https://openebs.github.io/dynamic-localpv-provisioner
+  version: 4.0.0
+- name: zfs-localpv
+  repository: https://openebs.github.io/zfs-localpv
+  version: 2.5.0
+- name: lvm-localpv
+  repository: https://openebs.github.io/lvm-localpv
+  version: 1.5.0
+- name: mayastor
+  repository: https://openebs.github.io/mayastor-extensions
+  version: 0.0.0
```

Comment:
Why 0.0.0?

Reply:
This file is not needed, would remove it.